bolt-modules/out/lib/puppet/functions/out/verbose.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+# Output a message for the user when running with --verbose
+#
+# This will print a message to stdout when using the human output format,
+# and print to stderr when using the json output format
+#
+# > **Note:** Not available in apply block
+Puppet::Functions.create_function(:'out::verbose') do
+  # Output a message when run with --verbose.
```

Comment:
```suggestion
```

Since the function has a single dispatch, the dispatch doesn't need a separate description (it'll be unused when our docs are generated).